### DISCUSSION TITLE
When will the first major version of vanilla-extract

### DISCUSSION THREAD

**Discussion by zainul-franciscus-myob** [vanilla-extract-css/vanilla-extract#79] [Q&A] [answered]
May we know when will the first major release be available as in vanilla-extract v1.0.0

**Comment by mattcompiles** [comment, marked as answer]
We're not going to commit to specific date as we don't know what issues could arise or last minute changes we may decide to make. Making vanillla-extract v1 will mean the API is completely stable and don't want to do that pre-maturely. 

However, we're currently aiming to do it in the next few weeks. 
☝️ Not a commitment.

**Reply by zainul-franciscus-myob** [reply]
Thank you
